index 5270661..f7ffd1b 100644
--- a/seafile-server/scripts/setup-seafile-mysql.py
+++ b/seafile-server/scripts/setup-seafile-mysql.py
@@ -1384,7 +1384,7 @@ def check_params(args):
     ccnet_config.server_name = ccnet_config.validate_server_name(server_name)
 
     server_ip = get_param_val(args.server_ip, 'SERVER_IP', '127.0.0.1')
-    ccnet_config.ip_or_domain = ccnet_config.validate_server_ip(server_ip)
+    ccnet_config.ip_or_domain = server_ip
 
     fileserver_port = get_param_val(args.fileserver_port, 'FILESERVER_PORT', '8082')
     seafile_config.fileserver_port = Utils.validate_port(fileserver_port)
