index 28e173c..ffc96c5 100644
--- a/ccnet-server/tools/ccnet-init.c
+++ b/ccnet-server/tools/ccnet-init.c
@@ -155,7 +155,7 @@ make_configure_file (const char *config_file)
 
     fprintf (fp, "[General]\n");
     if (host_str)
-        fprintf (fp, "SERVICE_URL = http://%s:8000\n", host_str);
+        fprintf (fp, "SERVICE_URL = http://%s\n", host_str);
 
     fprintf (fp, "\n");
     fclose (fp);
